R/stan_betareg.fit.R

```diff
@@ -24,7 +24,8 @@ stan_betareg.fit <- function(x, y, z = NULL,
                              link.phi = NULL, ...,
                              prior = normal(), prior_intercept = normal(),
                              prior_z = normal(), prior_intercept_z = normal(),
-                             prior_ops = prior_options(), prior_PD = FALSE, 
+                             prior_dispersion = cauchy(0, 5),
+                             prior_ops = NULL, prior_PD = FALSE, 
```

Comment:
`prior_ops` should be removed entirely instead of changed to `prior_ops=NULL`. The only reason it hasn't been removed entirely from `stan_glm.fit` is to maintain backwards compatibility, but there's no backwards compatibility to maintain for `stan_betareg` because it's new. So we should just never introduce `prior_ops` at all.